docs/source/release/v4.1.0/muon.rst

```diff
@@ -10,4 +10,8 @@ MuSR Changes
     improvements, followed by bug fixes.
 
 
-:ref:`Release 4.1.0 <v4.1.0>`
\ No newline at end of file
+:ref:`Release 4.1.0 <v4.1.0>`
+
+New
+###
+* Added phase tab for calculating phase tables and phase quad workspaces to Frequency Domain Analysis GUI.
```

Comment:
phasequad is one word and we should link to the documentation

Reply:
Done